app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] Kindly check/investigate for the reason this error happening on the page load(Hint: _add a root_)
![image](https://user-images.githubusercontent.com/62705947/207642684-01db6e45-38ef-433f-ad3c-a57c39d61227.png)

Reply:
Thank you @GzimAsani, could you please show me how to reproduce this error? :thinking: Because I have started the server on my side and it is working correctly

Reply:
I don't know if this might fix the issue on your side, but before running the server could you try:

```
1. yarn install
2. npm install
3. npm run build
```
Finally, start the server by running `./bin/dev`
Please let me know if it works :pray:

Reply:
- Hi @karlavdelgadof, I was getting an error, only on the root page. I did all those commands, give me 1 minute to try again, maybe it was a problem on my computer or something